scripts/UpdateTemplates.ps1

```diff
@@ -1,284 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we check-in this script in test-templates repo? We will have to create  nuget updations PRs there, this script can help us automate that! Though script will need some modifications as per the structure of the new repo.

Reply:
Not required. VS code works better.